src/view3d.cpp

```diff
@@ -535,7 +535,8 @@ Imath::V3d View3D::guessClickPosition(const QPoint& clickPos)
     // position x,y and the z of the reference position.  Take the reference to
     // be the camera rotation center since that's likely to be roughly the
     // depth the user is interested in.
-    V3d refPos = m_camera.center();
+//    V3d refPos = m_camera.center();
```

Comment:
Let's remove this commented out line.  The comment above also needs some editing since it's getting out of date.  How about:

```C++
// Get new point in the projected coordinate system using the click
// position x,y and the z of a reference position.  Take the reference point
// of interest to be between the camera rotation center and the camera
// position, as a rough guess of the depth the user is interested in.
//
// This works pretty well, except when there's noise points intervening
// between the reference position and the user's actual point of interest.
```